cves/CVE-2013-0847.yml

```diff
@@ -31,7 +31,7 @@ reported_instructions: |
   you can't find it out.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: ""
```

Comment:
Is it necessary to put the double quotes here? I understand that you could not find a reported date, but would it not follow YAML conventions to leave the quotes off?